src/resources/help/en_US/relnotes3.2.0.html

```diff
@@ -94,7 +94,8 @@
 <a name="Build_features"></a>
 <p><b><font size="4">Build features added in version 3.2</font></b></p>
 <ul>
-  <li>Build Feature 1</li>
+  <li>For build_visit, there is now only one way to specify a debug build: --build-mode "Debug".
+The --debug flag was removed.  The default build-mode is "Release".  If "Debug" is specified, all third-party libraries are built in debug mode.  If the intention is only to build VisIt in debug mode, it should be compiled separately after all the third-party libraries are compiled. Pass -DCMAKE_BUILD_TYPE:STRING=Debug to cmake when configuring VisIt.</li> 
```

Comment:
There is an extra space before "The" and the two "If"s.

Reply:
You might also change combine the last 2 sentences by changing "compiled. Pass" to "compiled, passing".

Reply:
About the spaces: a long time ago in a galaxy far far away, when I was a secretary typing documents, two spaces between sentences was mandatory.  Old (ancient?) habits die hard.